### PR TITLE
Remove my story into teaching section

### PR DESCRIPTION
**Pull request by peteryates**
**This PR follows on from #2563**

### Trello card

https://trello.com/c/MYSl7HO4/3267-remove-my-story-and-integrate-into-the-blogs

### Context

Some of the 'my story into teaching' posts have been moved to the blog (#2563) and the section can be decomissioned.

### CODE REVIEW

**Comment by gemmadallmandfe** [comment]
Need to update the link on the 404 page @peteryates - currently still going to /my-story-into-teaching?

File: app/views/content/train-to-be-a-teacher/if-you-want-to-change-career.md [hunk 1]
```diff
@@ -4,8 +4,8 @@ heading: If you want to change career
 description: |-
   Find out what extra support you can get if you want to change career and get into teaching. Bring your experience to life in the classroom.
 related_content:
-    From construction site to classroom : "/my-story-into-teaching/career-changers/from-construction-site-to-classroom"
-    Swapping senior management for students : "/my-story-into-teaching/career-changers/swapping-senior-management-for-students"
+    From engineering to teaching physics: "/blog/natural-transition-from-engineering-to-teaching-physics"
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/if-you-want-to-change-career.md:7]: the order of these links seems a little weird - two links re engineering to physics but separated?

one for @EllieNodder !

**Review by gemmadallmandfe — COMMENTED** [review]
Looks good - a few minor changes but otherwise this should be good to go on Monday!

File: app/views/content/non-uk-teachers/return-to-england-after-teaching-overseas.md [hunk 2]
```diff
@@ -5,8 +5,6 @@ backlink: "../../"
 description: |-
   Return to England after teaching overseas and bring your skills and experience back to an English classroom. Get support to make your transition easier.
 related_content:
-    Returning to teaching with international experience : "/my-story-into-teaching/international-career-changers/returning-to-teaching-with-international-experience"
```
[the remark this answers is not in the document]
Comment (gemmadallmandfe) [reply app/views/content/non-uk-teachers/return-to-england-after-teaching-overseas.md:8]: @peteryates Can you change this for me? Thanks!

**Comment by gemmadallmandfe** [comment]
Thanks @peteryates - those two changes look good to me

Happy to approve this once @EllieNodder has cast her eye over it on Monday

File: app/views/content/train-to-be-a-teacher/if-you-want-to-change-career.md (hunk 1 above)
Comment (EllieNodder) [reply app/views/content/train-to-be-a-teacher/if-you-want-to-change-career.md:7]: I'd agree, and I'd also say the link text needs rewording - don't feel like they're differentiated enough

Comment (peteryates) [reply app/views/content/train-to-be-a-teacher/if-you-want-to-change-career.md:7]: Should we drop one of them?

File: app/views/content/train-to-be-a-teacher/if-you-want-to-change-career.md [hunk 3]
```diff
@@ -4,9 +4,9 @@ heading: If you want to change career
 description: |-
   Find out what extra support you can get if you want to change career and get into teaching. Bring your experience to life in the classroom.
 related_content:
-    From construction site to classroom : "/my-story-into-teaching/career-changers/from-construction-site-to-classroom"
-    Swapping senior management for students : "/my-story-into-teaching/career-changers/swapping-senior-management-for-students"
-    Engineers teach physics : "/subjects/engineers-teach-physics"
+    Changing career from engineering to teaching physics: "/blog/natural-transition-from-engineering-to-teaching-physics"
+    Engineers teach physics teacher training  courses : "/subjects/engineers-teach-physics"
```
Comment (peteryates) [inline app/views/content/train-to-be-a-teacher/if-you-want-to-change-career.md:8]:
```suggestion
    Engineers teach physics teacher training courses : "/subjects/engineers-teach-physics"
```

**Review by gemmadallmandfe — DISMISSED** [review]
Looks good to me

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**